pytensor/tensor/rewriting/math.py

```diff
@@ -423,6 +423,108 @@ def local_sumsqr2dot(fgraph, node):
                     return [new_out]
 
 
+@register_specialize
+@node_rewriter([mul, true_div])
+def local_mulexp2expadd(fgraph, node):
```

Comment:
nitpick, the other rewrites in the codebase follow a `word_word` and `_to_` convention, so this would be `local_mul_exp_to_exp_add`.